js/utility.js

```diff
@@ -10,4 +10,17 @@ export function unhideElement(elem) {
 
 export function isElementHidden(elem) {
    return elem.classList.contains(HIDDEN_CLASS_NAME);
+}
+
+export function hidingRoomElements() {
```

Comment:
Let's not link this utility functions to specific class

I would go for the following:

```suggestion
export function hideELements(array) {
   const elements = document.querySelectorAll(array);
   
   for (let i = 0; i < array.length; i++) {
        hideEleemnt(array[i]);
    }
}
```